include/zephyr/drivers/clock_control/hpmicro_clock_control.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2022 hpmicro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_CLOCK_CONTROL_HPMICRO_CLOCK_CONTROL_H_
+#define ZEPHYR_INCLUDE_DRIVERS_CLOCK_CONTROL_HPMICRO_CLOCK_CONTROL_H_
+
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/dt-bindings/clock/hpmicro-clocks.h>
+
+/**
+ * @brief Generate clock setting preset
+ *
+ */
+#define HPM_MAKE_SYSCTL_PRESENT(x)		(1 << x)
```

Comment:
This seems to be identical to the `BIT` macro?

Reply:
The meaning is different, the form is the same. I replaced BIT with.

Reply:
technically speaking, it should be `(1 << (x))`, but I think the point is why not just 
```
#define HPM_MAKE_SYSCTL_PRESENT(x)		BIT(x)
```